src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -1,58 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <!-- rev'ed to 4.0.1.0 even though 4.0.0.0 never shipped so that we can drop pre-release down to beta -->
-    <ProjectGuid>{4C2655DB-BD9E-4C86-83A6-744ECDDBDF29}</ProjectGuid>
-    <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'net47'">true</IsPartialFacadeAssembly>
-    <RunApiCompat Condition="'$(TargetGroup)' != 'netcoreapp' AND '$(TargetGroup)' != 'uap'">false</RunApiCompat>
-    <!-- We've explicitly authored a reference facade so we do not need to use the desktop implementation as ref -->
-    <PackageDesktopAsRef>false</PackageDesktopAsRef>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);net47;netcoreapp2.0</TargetFrameworks>
```

Comment:
There are ApiCompat failures due to having NetFrameworkMinimum and net47, but I'm unsure how to resolve them:

```

API compatibility errors between 'lib/net47/System.ValueTuple.dll' (system.valuetuple.4.5.0.nupkg)
 and 'lib/net462/System.ValueTuple.dll' (System.ValueTuple.4.6.0-ci.24572.1.nupkg): 
CP0003: [Baseline] lib/net47/System.ValueTuple.dll assembly public key token 'cc7b13ffcd2ddd51'
 does not match with lib/net462/System.ValueTuple.dll '31bf3856ad364e35'. 

```